core/jazz_is-slack-channel-available/components/error-handler.js

```diff
@@ -0,0 +1,43 @@
+/**
+	Nodejs Template Project
```

Comment:
header- please update header in all the files that are missing it.

Reply:
Included header